app/views/admin/users/_actions.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="flex relative justify-between s:justify-end gap-2 my-2 s:my-0">
```

Comment:
**Some** of these buttons inside that dropdown need to trigger appropriate modals... Currently they do nothing :)

@nickytonline I'd appreciate your help here. These can be empty modals for now and I'll move cheese around later on and style appropriately.

Reply:
I've got a proof of concept on my local at the moment. We can chat about it on Monday.